tests/components/fronius/test_sensor.py

```diff
@@ -122,10 +122,10 @@ def assert_state(entity_id, expected_state):
     assert_state("sensor.smart_meter_63a_energy_real_consumed", 15303334)
     assert_state("sensor.smart_meter_63a_energy_real_produced", 35623065)
     assert_state("sensor.smart_meter_63a_frequency_phase_average", 50)
-    assert_state("sensor.smart_meter_63a_power_apparent_phase_1", 1772.793)
-    assert_state("sensor.smart_meter_63a_power_apparent_phase_2", 1527.048)
-    assert_state("sensor.smart_meter_63a_power_apparent_phase_3", 2333.562)
-    assert_state("sensor.smart_meter_63a_power_apparent", 5592.57)
+    assert_state("sensor.smart_meter_63a_apparent_power_phase_1", 1772.793)
```

Comment:
Wow that's a lot of asserts...
You should look into `SnapshotAssertion` as a follow-up PR :)